packages/common/src/system/retry-helper.spec.ts

```diff
@@ -44,20 +44,21 @@ describe(RetryHelper, () => {
     });
 
     it('success on retry', async () => {
-        let retried = false;
+        let retryCount = 0;
         actionMock
             .setup(a => a())
             .returns(async () => {
-                if (!retried) {
-                    retried = true;
+                if (retryCount < 2) {
+                    retryCount += 1;
                     throw testError;
                 }
 
                 return returnValue;
             })
-            .verifiable(Times.exactly(2));
-        errorHandlerMock.setup(e => e(testError)).verifiable(Times.once());
-        sleepFunctionMock.setup(s => s(It.isAny())).verifiable(Times.once());
+            .verifiable(Times.exactly(3));
+        errorHandlerMock.setup(e => e(testError)).verifiable(Times.exactly(2));
+        sleepFunctionMock.setup(s => s(millisBetweenRetries)).verifiable(Times.once());
+        sleepFunctionMock.setup(s => s(millisBetweenRetries * 2)).verifiable(Times.once());
 
         const result = await testSubject.executeWithRetries(actionMock.object, errorHandlerMock.object, maxAttempts, millisBetweenRetries);
```

Comment:
milliSecondsBetweenRetries

Reply:
millisecond is a single word, retryIntervalMilliseconds